modules/installing-private-image-registry-private-azure.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+//* registry/configuring_registry_storage-azure.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-private-image-registry-private-azure"]
+= Optional: Preparing a private Microsoft Azure cluster for a private image registry
+
+By installing a private image registry on a private Microsoft Azure cluster, you can create private endpoints. Private endpoints disable public facing endpoints to the internal registry, adding an extra layer of security to your {product-title} deployment.
+
+Use the following guide to prepare your private Microsoft Azure cluster for installation with a private image registry.
+
+.Prerequisites
+
+* You have access to an {product-title} account with cluster administrator access.
+
+* You have installed the OpenShift CLI (oc).
+
+* You have prepared an `install-config.yaml` that includes the following information:
+** A file that includes your VPC configuration
+** The `publish` field is set to `Internal`
+** The `credentialsMode` field is set to `Manual`
+
+* You have created a global Microsoft Azure account for the `ccoctl` utility.
+
+.Procedure
+
+. In your `install-config.yaml` file, add the following permissions that are required for creating an image registry storage account for a private Azure cluster:
++
+[source,yaml]
+----
+# ...
+credentialsMode: Manual
+# ...
+publish: Internal
+# ...
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AzureProviderSpec
+    permissions:
+      - Microsoft.Storage/storageAccounts/blobServices/read
+      - Microsoft.Storage/storageAccounts/blobServices/containers/read
+      - Microsoft.Storage/storageAccounts/blobServices/containers/write
+      - Microsoft.Storage/storageAccounts/blobServices/generateUserDelegationKey/action
+      - Microsoft.Storage/storageAccounts/read
+      - Microsoft.Storage/storageAccounts/write
+      - Microsoft.Storage/storageAccounts/delete
+      - Microsoft.Storage/storageAccounts/listKeys/action
+      - Microsoft.Resources/tags/write
+      - Microsoft.Network/privateEndpoints/write
+      - Microsoft.Network/privateEndpoints/read
+      - Microsoft.Network/privateEndpoints/privateDnsZoneGroups/write
+      - Microsoft.Network/privateEndpoints/privateDnsZoneGroups/read
+      - Microsoft.Network/privateDnsZones/read
+      - Microsoft.Network/privateDnsZones/write
+      - Microsoft.Network/privateDnsZones/join/action
+      - Microsoft.Network/privateDnsZones/A/write
+      - Microsoft.Network/privateDnsZones/virtualNetworkLinks/write
+      - Microsoft.Network/privateDnsZones/virtualNetworkLinks/read
+      - Microsoft.Network/virtualNetworks/subnets/join/action
+      - Microsoft.Network/virtualNetworks/join/action
+      - Microsoft.Network/networkInterfaces/read
+      - Microsoft.Storage/storageAccounts/PrivateEndpointConnectionsApproval/action
```

Comment:
I think those permissions should not be listed in the doc, but only adding into registry credential request manifest, which will be consumed when creating separated credentials if cluster is configured in credentialsMode: Manual mode.

Reply:
@jinyunma To be sure, you're requesting that I remove the permissions from this procedure? I have updated the PR to remove this step. 

If an alternative step is required, would you mind providing it?

Reply:
I think no alternative step is required, cc @flavianmissi and @wewang58 to check or any other concern?

Reply:
Yes no alternative step is required